crates/bevy_math/src/ops.rs

```diff
@@ -79,7 +79,8 @@ mod std_ops {
     }
 
     /// Compute the distance between the origin and a point `(x, y)` on the Euclidean plane.
-    /// Equivalently, compute the length of the hypotenuse of a right-angle triangle with other sides having length `x.abs()` and `y.abs()`.
+    ///
+/// Equivalently, compute the length of the hypotenuse of a right-angle triangle with other sides having length `x.abs()` and `y.abs()`.
```

Comment:
Indentation here looks wrong.

Reply:
Yup, that was clippy's doing and I didn't run `cargo fmt` enough times. ><